src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,65 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Arrays;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private final int arrayLength = 10;
+    @SuppressWarnings("unchecked") private final K[] keyArray = (K[]) new Object[arrayLength];
+    @SuppressWarnings("unchecked") private final V[] valueArray = (V[]) new Object[arrayLength];
+    private int arraySize;
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < keyArray.length; i++) {
+            if ((Objects.equals(keyArray[i], key)) && (valueArray[i] != null)) {
```

Comment:
```suggestion
            if (Objects.equals(keyArray[i], key)) {
```
тут такої перевірки має бути достатньо